dev_tools/rubocop/infrastructure.yml

```diff
@@ -1,8 +1,5 @@
-inherit_from: base.yml
-
-require:
-  - rubocop-performance
-  - rubocop-rspec
+inherit_from:
+  - base.yml
 
 Bundler/OrderedGems:
   Enabled: false
```

Comment:
This is just personal preference but I actually prefer having this enabled (the default) and using newlines to separate gems into logical groups.

The gems in each newline group are then sorted alphabetically.

[core example](https://github.com/generalassembly/core/blob/master/Gemfile#L226)

I made a mistake with the `webmock`/`vcr` group though.

Reply:
Oh, so it requires sorting within chunks (e.g., not across newlines)? I'm down for that.

Reply:
Lets figure this out in another PR. Not looking to change much in the way of style here :)